src/sentry/static/sentry/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -195,50 +196,56 @@ function ChangedTransactions(props: Props) {
                   <QuestionTooltip size="sm" position="top" title={titleTooltipContent} />
                 </HeaderTitleLegend>
               </ContainerTitle>
-              {transactionsList.length ? (
+              {isLoading ? (
+                <LoadingIndicator />
```

Comment:
Do you need to set a height to prevent layout shift?

Reply:
Yeah definitely should 👍